### DISCUSSION TITLE
HW1 Install Instructions for GameCube

### DISCUSSION THREAD

**Discussion by Colrut** [darthcloud/BlueRetro#808] [Show and tell]
Attached are step-by-step instructions for installing internal HW1 spec for GameCube.  Includes LED indicator light and use of reset switch to control the ESP32 board.

Enjoy

[BlueRetro Install.pdf](https://github.com/darthcloud/BlueRetro/files/13030972/BlueRetro.Install.pdf)

**Comment by darthcloud** [comment]
Great work! Thanks for contributing instructions.
